app/server/src/controllers/sessionsController.ts

```diff
@@ -56,4 +56,12 @@ export class SessionsController {
         const session = await store.getSession(id);
         serialiseSession(session, res);
     };
+
+    static generateFriendlyId = async (req: Request, res: Response) => {
+        const { redis, wodinConfig } = req.app.locals as AppLocals;
+        const { appName, id } = req.params;
+        const store = new SessionStore(redis, wodinConfig.savePrefix, appName);
```

Comment:
Looks like this snippet could be extracted into it's own method and reuse in `GetSession` and `postSessionLabel` methods.